.github/workflows/build.yml

```diff
@@ -36,7 +34,7 @@ jobs:
         uses: docker/setup-buildx-action@v3
 
       - name: Docker Login
-        if: success() && github.event_name != 'pull_request' && github.repository == 'photoview/photoview'
+        if: ${{ env.IS_PUSHING_IMAGES == 'true' }}
```

Comment:
the `== 'true'` part is useless, as the variable already contains `true` or `false`

Reply:
Nope. The ENV value is a `true` string, but not a boolean. `if` requires a boolean. So here we need the comparison.

I tried without the comparison before, and it fails.